services/app/src/components/base/Dialogs.tsx

```diff
@@ -111,7 +111,10 @@ export class ExpansionSelectDialog extends React.Component<ExpansionSelectDialog
           <Button className="primary large" onClick={() => this.props.onExpansionSelect({horror: false})}>Expedition</Button>
           <br/>
           <br/>
-          <Button className="primary large" onClick={() => this.props.onExpansionSelect({horror: true})}><span className="line">Expedition&nbsp;</span><span className="line">+ The Horror</span></Button>
+          <Button className="primary large" onClick={() => this.props.onExpansionSelect({horror: true})}>Expedition + <img className="inline_icon" src="images/horror_small.svg"/></Button>
```

Comment:
I'm not sure if icons are the most understandable thing to see when a new user opens the app for the first time... what does this look like rendered?

Reply:
(see screenshot and idea in main comment)